srv/GetCartesianPath.srv

```diff
@@ -43,6 +43,12 @@ bool avoid_collisions
 # Specify additional constraints to be met by the Cartesian path
 Constraints path_constraints
 
+# Maximum cartesian speed for the given end effector.
+# If max_cartesian_speed <= 0 the trajectory is not modified.
+# These fields require the following planning request adapter: default_planner_request_adapters/SetMaxCartesianEndEffectorSpeed
```

Comment:
The Cartesian service in `move_group` does not use the planning pipeline, so believe this statement is wrong.

Reply:
Yes you are right. Too much copy and paste there.